frontend/components/common/Icon.js

```diff
@@ -26,7 +26,7 @@ export const Icon = ({ name, size = '1rem', ...props }) => {
   return (
     <IconWrapper>
       {svg.default({
-        preserveAspectRatio: 'true',
+        preserveAspectRatio: 'xMidYMid meet',
```

Comment:
Wat this do?

Reply:
Centers svg icons in its container, while maintaining its aspect ratio